### PR TITLE
fix: extend expiration date

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [] 

### Description

Change Limited-Time offer expiration date from today to in a year 🤭 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Change Limited-Time offer expiration date from today to in a year - daria

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/Home/Components/ActivityRailItem.tests.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ describe("ActivityRail", () => {
           item: {
             __typename: "PartnerOfferCreatedNotificationItem",
             available: true,
-            expiresAt: "2024-11-11T11:11:11+00:00",
+            expiresAt: "2025-11-11T11:11:11+00:00",
```
Comment (gkartalis) [inline src/app/Scenes/Home/Components/ActivityRailItem.tests.tsx:98]: @MrSltun fixed this [here](https://github.com/artsy/eigen/pull/11084) so we don't care anymore about the expiresAt, but we "mock" today `Date.now = () => 1730459471000` by telling it it is not expired

Comment (gkartalis) [reply src/app/Scenes/Home/Components/ActivityRailItem.tests.tsx:98]: I did remove the #squashOnGreen label due to the reason above @dariakoko

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Change Limited-Time offer expiration date from today to in a year - daria - dariakoko)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e28de6e86842226bc43acf700cc04e154ced7dcd
</p>

File: src/app/Scenes/Home/Components/ActivityRailItem.tests.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/Home/Components/ActivityRailItem.tests.tsx:98]: Alright, thanks!

**Pull request closed without merging**